extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilderSuite.scala

```diff
@@ -155,15 +161,19 @@ abstract class PrivilegesBuilderSuite extends AnyFunSuite
   }
 
   test("CreateDatabaseCommand") {
+    assume(isSparkVersionAtMost("3.2"))
```

Comment:
what's wrong with these test after such a change?

Reply:
We add catalog assertion in thi ut.

In spark 3.0-3.2,  `CREATE DATABASE` sql is parsed into `CreateDatabaseCommand`, which is v1 command and contains no catalog info . And this ut asserts `catalog` in  parsed PviilegeObject is None.

 In spark 3.3, the same ddl will is pared into `CreateNamespace` of v2commands instead, and the asserted catalog is extracted by another related ResolvedDBObjectNameDatabaseExtractor with real value from CatalogPlugin. New ut for this case is also added to `V2CommandsPrivilegesSuite.scala` in this PR.

Reply:
in this case, we shall not put a uncommon test case in the base trait.

Reply:
I think they are actually testing different command plans and they are good enough to reflect spark version variation impact for this ddl.

Reply:
Plus, the migration from v1 to v2 is generally refactoring command detail itself affecting same commands to all and not related to v2 plans.

Reply:
thanks for the explanation. I want to eliminate spark version and v1/v2 table testing, maybe it's too early to do this.